package.json

```diff
@@ -1,13 +1,17 @@
 {
-  "name": "buefy",
-  "version": "0.9.23",
+  "name": "buefy-next",
```

Comment:
@wesdevpro Doesn't this need to be "@ntohq/buefy-next"? I may be wrong again.

Reply:
> @wesdevpro Doesn't this need to be "@ntohq/buefy-next"? I may be wrong again.

No it shouldn't because the actual package name is buefy-next. It is the same thing for @buefy/buefy. Basically, NPM will figure out that buefy-next needs to publish to @ntohq/buefy-next via this registry link specified in the workflow: `https://registry.npmjs.org/ntohq`

Reply:
> If you're publishing a package that includes a scope prefix, include the scope in the name of your package.json file. For example, if your npm scope prefix is "octocat" and the package name is "hello-world", the name in your package.json file should be @octocat/hello-world. If your npm package uses a scope prefix and the package is public, you need to use the option npm publish --access public. This is an option that npm requires to prevent someone from publishing a private package unintentionally.

@kikuomax you may be correct. Strange.